### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by goldenuni**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,10 +1,49 @@
...
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for nickname, info in data.items():
+        race = info.get("race")
+        if not Race.objects.filter(name=race.get("name")).exists():
+            Race.objects.create(
```
[the remark this answers is not in the document]
Comment (ArsenPidhoretskyi) [reply main.py:14]: Fix the same with Guild and skill

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: main.py [hunk 2]
```diff
@@ -1,10 +1,43 @@
...
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for nickname, info in data.items():
+        race = info.get("race")
+        race_id, _ = Race.objects.get_or_create(
```
Comment (ArsenPidhoretskyi) [inline main.py:13]:
```suggestion
        race, _ = Race.objects.get_or_create(
```

File: main.py [hunk 3]
```diff
@@ -1,10 +1,43 @@
...
+            name=race.get("name"),
+            description=race.get("description")
+        )
+
+        skills = race.get("skills")
+        for skill in skills:
+            skill_id, _ = Skill.objects.get_or_create(
```
Comment (ArsenPidhoretskyi) [inline main.py:20]:
```suggestion
            Skill.objects.get_or_create(
```

File: main.py [hunk 4]
```diff
@@ -1,10 +1,43 @@
...
+                name=skill.get("name"),
+                bonus=skill.get("bonus"),
+                race=Race.objects.filter(name=race.get("name")).get()
+            )
+        if info.get("guild"):
+            guild = info.get("guild")
+            guild_id, _ = Guild.objects.get_or_create(
```
Comment (ArsenPidhoretskyi) [inline main.py:27]:
```suggestion
            guild, _ = Guild.objects.get_or_create(
```

Comment (ArsenPidhoretskyi) [reply main.py:27]: guild_id is meaning as int

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: main.py (hunk 4 above)
Comment (goldenuni) [reply main.py:27]: okay)

**Pull request closed without merging**